jobs/generation/Utilities.groovy

```diff
@@ -300,9 +304,10 @@ class Utilities {
     //  job: Job to set up.
     //  project: Name of project
     //  isPR: True if job is PR job, false otherwise.
-    //  defaultBranch: If not a PR job, the branch that we should be building.
-    def static standardJobSetup(def job, String project, boolean isPR, String defaultBranch = '*/master') {
-        Utilities.addStandardParameters(job, project, isPR, defaultBranch)
+    //  gitBranchOrCommit: Commit / branch to build.  Defaults to */master for push jobs, and ${sha1} for PR jobs.
+    //  gitRefSpec: the refs that Jenkins must sync on a PR job.
+    def static standardJobSetup(def job, String project, boolean isPR, String gitBranchOrCommit = null, String gitRefSpec = DefaultGitRefSpec) {
```

Comment:
Actually, can you make this symmetric with respect to the gitBranchOrCommit = null and the gitRefSpec (should be null), then a method: getDefaultRefSpec().  This is mainly in case we ever seen to have another default ref spec.